### PR TITLE
Feat/reset pnps

### PR DESCRIPTION
**Pull request by FredrikOseberg**
* This PR resets pnps prompts by removing entries in the user_feedback table

### CODE REVIEW

**Review by ivarconr — APPROVED** [review]

File: src/migrations/20230927071830-reset-pnps-feedback.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+       DELETE FROM user_feedback WHERE feedback_id = 'pnps';
```
Comment (ivarconr) [inline src/migrations/20230927071830-reset-pnps-feedback.js:6]: Maybe only when it is older than 3 months?

**Review by ivarconr — CHANGES_REQUESTED** [review]

File: src/migrations/20230927071830-reset-pnps-feedback.js (hunk 1 above)
Comment (FredrikOseberg) [reply src/migrations/20230927071830-reset-pnps-feedback.js:6]: Unfortunately, this table does not have any timestamp columns.

Comment (sjaanus) [reply src/migrations/20230927071830-reset-pnps-feedback.js:6]: There is column `given`, with timestamp.

Comment (FredrikOseberg) [reply src/migrations/20230927071830-reset-pnps-feedback.js:6]: > There is column `given`, with timestamp.

Yes, but that is only populated if you gave feedback. If you hide the feedback form it won't be populated.

Comment (FredrikOseberg) [reply src/migrations/20230927071830-reset-pnps-feedback.js:6]: So two options: 
* We clear all entries
* We clear only the entries that have previously given feedback

What do you prefer?

**Review by ivarconr — APPROVED** [review]

**Pull request merged**